rfcs/2021-09-21-9292-throttle-transform.md

```diff
@@ -0,0 +1,153 @@
+# RFC 9292 - 2021-09-21 - `throttle` transform
+
+This RFC proposes the addition of a new transform that provides a user the ability to control the throughput of specific event streams.
+
+## Context
+
+- [#258](https://github.com/vectordotdev/vector/issues/258)
+
+## Scope
+
+### In scope
+
+- Dropping events (logs and metrics) to rate limit an event stream
+- Rate limit by both bytes (unserialized) and events
+- Exclude events with VRL conditions
+- Specify buckets based on the value of a key in the event
```

Comment:
Or template, I'd say.

Reply:
🤔 same difference? I was thinking along the lines of [`key_field`](https://vector.dev/docs/reference/configuration/transforms/sample/#key_field).